lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -310,7 +310,7 @@ def get(self, request, *args, **kwargs):
                         # another type, just skip it (don't filter it out).
                         seq_data['type'] != 'sequential'
                     )
-                ]
+                ] if 'children' in chapter_data else []
```

Comment:
This was unrelated. Jansen noticed this bug - if a sequence has no children, we'd throw an exception here on the MFE outline page.